CommandHelper/LocalPackages/commands/chars/joker.yml

```diff
@@ -4,6 +4,8 @@ costumes:
   name: §fJo§2k§5er
   sig: AwQXimwxx8rTzOYRYqOVy03JYdNhKcjcc9TpQj+CybBC1Nf/mKV5kPk5AMpxPxA+DPvf+bxt4vdC5tUn+mQ8hvRymcidxr8ST4Ztdy6EYsmo3seAzONko3tSq4v5ccoSwVTbnj/6Hp5XYJMkBEik7Szm7rnz55dvl/MHTnc6TQ2dto9siCUu1GX35AqsiQHBBQ6XlKPAdegEWL8783aSs542Inad0uvq2b3nG3cnP1PXES6qCLxPKciKf+lwasIIqFyU+3qXIDxw8CsXpISGhvj/JQv+Snn5sjvWK/vSlxm3W1FRAZJSYdHAK0HkF7gOnHuoKYQ7eEe6LVmelQ71h6t3PpvGBXaV6wnetnCN9zv86SbOhiotOT+7FW0KVext6uY7XfLYPQj+ggLsIhKzFPw6RiCH7Xj2J8b3Gwo9spUGcFvfzLUMIv37Nob1b1QQ2jU2MZFVvCJSh7TOrzdKFrge1rG49CULt8wtwPI6PfOEVi+iXOpGVQinu+ANBDuMbR9dI5bkuFQ42vn01e0bIgbPJb12GjBjiWRmaZEODlzt31/EgOJHzcaRi5dLaUz9TmLn4XeVkczqy1SVwNCAfn8G8b5HlPreGGCDEL2NaRE1BfHzQ1fbvhJ/NI41yuimbfOxidw4cnQuqmq42Vjxqry46lQK9b7/Nsx21k7MtYY=
   val: eyJ0aW1lc3RhbXAiOjE0NzQ4MjU4NjQ2NjcsInByb2ZpbGVJZCI6Ijg3ZmIzYzZjNWZmNjRiZjZiMzI2OTkyYzYxMzdlYTEyIiwicHJvZmlsZU5hbWUiOiJUZWt0cm9uaWMiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlL2RlZjZhOGYwNjU4MDQwZTk0NmI4NjViYmU1ODdjYzVkZjYwMzVkM2MwOWQ1NDNmYjI5MzZiZjI3YWJkMzE4In19fQ==
+  sig:T+yE5/CGfbkxZbxXsikBA2fiDIyfyYcUONeHkaLC1Zhm1t2OMlPU1EXLX962Wsy/4g517skzS4Uhsjro1U1RbY/5ZYmwUSzG4+usznkX+zNQ9rYxVcf21ZPjXHLC1RdjcPZnr21ocHaR9y89GeM2N+kKYcZ6EMG4kHBXJt1FMSicTGfpizFgYehCXVqf78I/s04bDukMRoiPQ8RoKXyJZn+bb1dXy9ihwngXTL67BvKHzt24qlnlFA9P4F+ptNEH8LdiMQ2V30s65Ku2Vew8gMwaGK5gmiUg5spZPP1kO4GV0Myr/eX0ZmRIgjR5kfpMPZhr7cpwfr9asEGJZPLAJLQv2b/HnsqZyXHt4KOb/jrqrxgeJHsajZifYJY1w3y4qTQG1Me3SQK+6S8Ziiwb6CKdzAjw45/Omv4eKmHvgkYFBORCKcSt+niqdFibF519fQ0Na0LNei+42QCjpSXRSATFaEpkniXfckUFaldPXmdE3qVMUeO+57X3nGhT8ZRPiTHDjcaxVg2vh0T7GzqP9XWelL1WkQDNb7lS8/8SWwfI7xfMO/dgqXxkZx8c3okBc9YJpzD7B6DFkAdpjF8pc0I05wnoY9gsOPtUiTANzdVHiMjizVcEVfGPDgeohf/e+jwOYoFmuAwFEUMqGPCEkn/lPOCJkgaz1lV6AxHXX68=
+  val:eyJ0aW1lc3RhbXAiOjE1MjMyMzc1MjU1NzYsInByb2ZpbGVJZCI6ImVmN2JjYzk2M2FjMDRkYzZhYWU3MDE5ZTVjNmRkYTJlIiwicHJvZmlsZU5hbWUiOiJzdXBlcmlvcnhtYW4iLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzNkYzU0MjU1NDg1YWQzMjRlZWU0ZTg3YjRmMzJmZGMwNjMyYzM4ZjQ1YWRlYjMzZTE2NWI2MWQxNzFhZjEzYiIsIm1ldGFkYXRhIjp7Im1vZGVsIjoic2xpbSJ9fX19
```

Comment:
requires space after sig and val

you didnt delete the old sig and val

why are you even changing the skin? whats wrong with the current skin? why not just add it as a new costume?